homeassistant/components/unifi/switch.py

```diff
@@ -51,8 +51,8 @@
 CLIENT_BLOCKED = (EventKey.WIRED_CLIENT_BLOCKED, EventKey.WIRELESS_CLIENT_BLOCKED)
 CLIENT_UNBLOCKED = (EventKey.WIRED_CLIENT_UNBLOCKED, EventKey.WIRELESS_CLIENT_UNBLOCKED)
 
-Data = TypeVar("Data")
-Handler = TypeVar("Handler")
+_DataT = TypeVar("_DataT")
+_HandlerT = TypeVar("_HandlerT")
```

Comment:
It's a shame that `Outlet` and `Port` do not inherit `APIItem`, and that `Outlets` and `Ports` do not inherit `APIHandler`.
Until they do, I suggest that you still add binding to the typevar:

```suggestion
_DataT = TypeVar("_DataT", bound=Union[APIItem, Outlet, Port])
_HandlerT = TypeVar("_HandlerT", bound=Union[APIHandler, Outlets, Ports])
```

Reply:
Thanks! Resolved.